_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -8963,22 +8963,22 @@ std::vector<ENCODE_PACKEDHEADER_DATA> const & HeaderPacker::PackSlices(
     DdiTask const & task,
     mfxU32          fieldId)
 {
+    const mfxU32 maxSliceHeaderSize = 50;
```

Comment:
Where did 50 come from? sizeof? at least explain in code.
Better use size_t for all types here and thus avoid all castings. Old style castings.